mitosheet/src/mito/Mito.tsx

```diff
@@ -305,7 +305,9 @@ export const Mito = (props: MitoProps): JSX.Element => {
                             }
                         }
                     })
+                    void mitoAPI.log('user_edited_code_cell')
```

Comment:
It would be really nice to get the params in this log call as well. Can we make the oldCode and newCode params to the callback function for opening the dialogue, and use them here as well?